01_prepare_host.sh

```diff
@@ -109,6 +109,28 @@ if [ ! -f "${IMAGE_NAME}" ] ; then
 fi
 popd
 
+# Local registry for images
+reg_state=$(sudo "$CONTAINER_RUNTIME" inspect registry --format  "{{.State.Status}}" || echo "error")
+if [[ "$reg_state" != "running" ]]; then
+ sudo "${CONTAINER_RUNTIME}" rm registry -f || true
+ sudo "${CONTAINER_RUNTIME}" run -d -p 5000:5000 --name registry "$DOCKER_REGISTRY_IMAGE"
+fi
+
+for IMAGE_VAR in $(env | grep "_IMAGE=" | grep -o "^[^=]*") ; do
+  IMAGE="${!IMAGE_VAR}"
+  #shellcheck disable=SC2086
+  export $IMAGE_VAR="${IMAGE##*/}:latest"
```

Comment:
this might not be correct, you would need to respect the tag, as it is important and defines the version you are using (for example v1a3 vs v1a4)